src/less/listbox-button/base/listbox-button.less

```diff
@@ -0,0 +1,67 @@
+@import "../../less/variables-common.less";
+@import "../../less/mixins-common.less";
+
+.listbox-button {
+    line-height: normal;
+    position: relative;
+}
+
+span.listbox-button {
+    display: inline-block;
+}
+
+div.listbox-button__listbox {
+    .dropdown;
+
+    border-bottom-color: transparent;
+    overflow: visible; // show focus outline
+}
+
+button.expand-btn[aria-expanded="true"] ~ div.listbox-button__listbox {
+    display: block;
+}
+
+// todo: refactor to use a button mixin rather than .expand-btn class
+.listbox-button button.expand-btn--borderless {
+    background-color: transparent;
+    border-color: transparent;
+    padding-left: 0;
+    vertical-align: initial;
+
+    &:focus {
+        outline: none;
+        text-decoration: underline;
+    }
+
+    &[aria-expanded="true"] {
+        border-bottom-color: transparent;
+    }
+
+    &[aria-expanded="true"] ~ .listbox-button__listbox {
```

Comment:
Can put this inside the previous section I believe

&[aria-expanded="true"] {
         ~ .listbox-button__listbox {
 // etc
        }
        border-bottom-color: transparent;
}

Reply:
Good spot. I'll keep that one in mind when I do a final sweep through again after Menu & Combobox PRs are done.